include/cartesian_geom/point.h

```diff
@@ -119,6 +119,16 @@ class point
         this->coeffs = coeffs + this->coeffs;
     }
 
+    void operator-= (const point& p)
+    {
+        coeffs -= p.getCoefficients();
+    }
+
+    void operator-= (const Coeff& coeffs)
+    {
+        this->coeffs = coeffs - this->coeffs;
```

Comment:
Is it possible to use the "-=" operator here as well (it is more efficient)

Reply:
I changed this.